src/cfnlint/core.py

```diff
@@ -146,14 +149,14 @@ def get_args_filenames(cli_args):
         sys.exit(0)
 
     if not sys.stdin.isatty() and not config.templates:
-        return(config, [None], formatter)
+        return (config, [None], formatter)
```

Comment:
Please don't mix functional changes with style changes :)

Reply:
Will keep in mind! I'm assuming you mean keep them in separate commits?